.devcontainer/devcontainer.json

```
@@ -30,7 +30,7 @@
 	// "forwardPorts": [],
 
 	// Use 'postCreateCommand' to run commands after the container is created.
-	// "postCreateCommand": "yarn install",
+	"postCreateCommand": "npm install -g typescript"
```

```suggestion
	"postCreateCommand": "npm install"
```

Instaling typescript compiler is never recommended allways use a local version with a specific venison